test/e2e/main/smoke_test.go

```diff
@@ -18,83 +22,79 @@ import (
 func TestNodeStartup(t *testing.T) {
 	defer testKube.CleanNamespaceAndLogOnPanic(t, tutils.Namespace)
 
-	// Create a resource without passing any config
+	// Infinispan cluster defintion with extra labels to be propagated to the service and pods
+	replicas := 2
```

Comment:
I think we should keep the `TestClusterFormation` name instead of `TestNodeStarupt` as the former implies pod startup and cluster creation works.

Reply:
Could be, I was actually thinking on who to name this one as it doesn't test only cluster formation but default authentication and labels too. Maybe `TestDefaultClusterFeatures`?

Reply:
Sounds good.